src/database/models/topic-assessment-info.ts

```diff
@@ -0,0 +1,153 @@
+import { Model, DataTypes } from 'sequelize';
+import appSequelize from '../app-sequelize';
+
+interface TopicAssessmentInfoInterface {
+    id: number;
+    courseTopicContentId: number;
+    duration: number;
+    hardCutoff: boolean;
+    maxGradedAttemptsPerVersion: number;
+    maxVersions: number;
+    versionDelay: number;
+    hideHints: boolean;
+    showItemizedResults: boolean;
+    showTotalGradeImmediately: boolean;
+    hideProblemsAfterFinish: boolean;
+    randomizeOrder: boolean;
+}
+
+export default class TopicAssessmentInfo extends Model implements TopicAssessmentInfoInterface {
+    public id!: number;
+    public courseTopicContentId!: number;
+    public duration!: number; // enforce IN MINUTES
+    public hardCutoff!: boolean;
+    public maxGradedAttemptsPerVersion!: number;
+    public maxVersions!: number;
+    public versionDelay!: number; // for consistency do we also force MINUTES here?
```

Comment:
I think so, should we append `minutes` to the field name to make it clear?

Reply:
ehhh, I think that's important for UI -- but what if we change to nanoseconds in the future?